lib/manage-prototype-handlers.js

```diff
@@ -283,18 +283,22 @@ function locateTemplateConfig (req) {
 }
 
 function getTemplatesViewHandler (req, res) {
-  const model = {
-    pluginConfig: plugins.getAppConfig({ scripts: prototypeAppScripts }),
-    serviceName: 'Service name goes here'
-  }
   const templateConfig = locateTemplateConfig(req)
 
   // Nunjucks environment for template previews uses `getAppViews()` to
   // add plugins including GOV.UK Frontend views via project package
   const nunjucksAppEnv = nunjucksConfiguration.getNunjucksAppEnv(appViews)
 
+  // Use GOV.UK Frontend paths from Express.js locals
+  const { govukFrontend, govukFrontendInternal } = req.app.locals
+
   if (templateConfig) {
-    res.send(nunjucksAppEnv.render(templateConfig.path, model))
+    res.send(nunjucksAppEnv.render(templateConfig.path, {
+      govukFrontend,
+      govukFrontendInternal,
```

Comment:
For template previews we use a minimal Nunjucks context only

But to correctly build `assetPath` we need GOV.UK Frontend paths to close https://github.com/alphagov/govuk-prototype-kit/issues/2392

https://github.com/alphagov/govuk-prototype-kit/blob/73e4b7d73ec71553c2cc7ddadf680d0fd4a1fcc2/lib/nunjucks/govuk-prototype-kit/layouts/govuk-branded.njk#L1

https://github.com/alphagov/govuk-prototype-kit/blob/73e4b7d73ec71553c2cc7ddadf680d0fd4a1fcc2/lib/nunjucks/views/manage-prototype/layout.njk#L1